views/index.html

```diff
@@ -16,7 +16,7 @@
         <div class="section-box">
           <div class="offering-icon mvp-icon">
           </div>
-          <h3>Product Development</h3>
+          <h3>Digital Product Development</h3>
```

Comment:
My comment for the business cards is that there was no contextual information whereas everything on the website is about tech so there's a lot more context.

Still works though 👍